static/_redirects

```diff
@@ -189,6 +189,7 @@
 /docs/home/contribute/review-issues/ /docs/contribute/intermediate/ 301
 /docs/home/contribute/blog-post/ /docs/contribute/start/ 301
 /docs/home/contribute/write-new-topic/ /docs/contribute/style/write-new-topic/ 301
+/docs/contribute/style/page-templates/ /docs/contribute/ 301
```

Comment:
Hi @jai .
I looked back to v1.17. If I recall correctly, the template page, ` /docs/contribute/style/page-templates/ `, was replaced by the content types page,  `/docs/contribute/style/page-content-types/`.

What do you think about updating line 187 to point to the `page-content types` page or to the main style page,
`/docs/contribute/style/`?

Line 192 could also redirect to `/docs/contribute/style` or `/docs/contribute/style/page-content-types`.

Reply:
@kbhawkey sure I'll push the commits soon, thanks for the comment!

Reply:
done!